modules/oe_content_publication/oe_content_publication.post_update.php

```diff
@@ -77,4 +148,14 @@ function oe_content_publication_post_update_00003() {
       ->updateFromStorageRecord($form_display, $form_display_values);
     $updated_form_display->save();
```

Comment:
We need to add config hash here too, or else field group labels won't be translatable.

Reply:
I added the config hash value.